src/main/openapi/problem/v1/problem-v1.yaml

```diff
@@ -56,13 +56,13 @@ components:
           description: An absolute URI that identifies the specific occurrence of the problem. It may or may not yield further information if dereferenced.
       example:
         {
-          "type": "urn:problem-type:belgif:payloadTooLarge",
-          "href": "https://www.belgif.be/specification/rest/api-guide/problems/payloadTooLarge.html", # location of linked doc will change in the future to recommended URI structure
-          "title": "Payload Too Large",
-          "status": 413,
-          "detail": "Request message must not be larger than 10 MB",
-          "instance": "urn:uuid:123e4567-e89b-12d3-a456-426614174000",
-          "limit": 10485760  # additional properties specific to the problem type are allowed
+          "type": "urn:problem-type:exampleOrganization:exampleProblem", # "exampleOrganization" should be a short identifier for the organization that defines the problem type. "belgif" is used for problem types standardized in the Belgif REST guide
```

Comment:
I remember a discussion about duplication of these comments of the example with the description of the properties. I think the idea was to align them?

Currently the example comments contain additional useful information, which is only visible when reading the source code. The description of the properties is more visible in tools like swagger-ui.

Reply:
It was only said not to duplicate the same documentation twice.
I pushed a new commit though, to put more in the descriptions.

The remaining documentation in code comments is meant for API providers, so it should be OK if it's hidden in SwaggerUI, e.g. structure of the problem type URN shouldn't be interpreted (parsed) by clients, and the explanation "location within the body" only applies to the "name" property in the example; the JSON path structure isn't a guideline.